apsi-enrollment-be/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM gradle:6.3-jdk11
+WORKDIR /usr/src/app
+COPY . .
```

Comment:
Czy do takich kopii nie chcemy zapewnić jakiegoś `.dockerignore` żeby mieć pewność, że nie przekopiujemy jakichś naszych folderów i plików, które powstają w trakcie developa (jak np folder `build`) i ew. mogę wpłynąć na proces budowania obrazu (np. jakiś cache?).

Reply:
Wydaje mi się, że takie rzeczy nie powinny iść do repo, więc to według mnie zbędna redundancja